keras_cv/layers/preprocessing/jittered_resize.py

```diff
@@ -0,0 +1,237 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+
+import keras_cv
+from keras_cv import layers
+from keras_cv.layers.preprocessing.base_image_augmentation_layer import (
+    BaseImageAugmentationLayer,
+)
+
+
+class JitteredResize(BaseImageAugmentationLayer):
+    """JitteredResize implements resize with scale distortion.
+
+    JitteredResize takes a three step approach to size-distortion based image
+    augmentation.  This technique is highly tuned for object detection pipelines.
+    The layer takes an input of images and bounding boxes, both of which may be ragged.
+    It outputs a dense image tensor, ready to feed to a model for training.
```

Comment:
Optional: I personally like the "returns" docstring section, especially when it gets rendered e.g. on TF.org docs

Reply:
lets do this in a follow up when we have our KPL fixit.  ( I believe after the 0.4 )